.github/actions/access-app/action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Access application'
+description: 'Access application running on AKS cluster'
+inputs:
+  resource-group-name:
+    description: 'Resource group where the infrastructure is deployed'
+    required: true
+  aks-cluster-name:
+    description: 'AKS cluster name'
+    required: true
```

Comment:
Reading the code, this looks like this action causes a bunch of outputs. When I read the name `Access application`, I think it's going to do something, not just output a bunch of stuff so that one could subsequently do something. If I am correct, then I suggest renaming this to "Output application access endpoints".

Reply:
Good point. Fixed in 2a41705.